ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -20,13 +20,15 @@ const AssetListItem = ({
   iconClassName,
   onClick,
   tokenAddress,
-  tokenSymbol,
+  primarySymbol,
```

Comment:
Also related, when conversionRate is undefined (hardcoded in getConversionRate):

![image](https://user-images.githubusercontent.com/4448075/122820887-c022b600-d2a1-11eb-825f-22778e8e0911.png)

Reply:
Re the last comment, the conversionRate should be null when not available? and I'm not seeing that behavior where some tokens are USD and others are with their symbols...?

Regarding the shape you suggest for primary/secondary -> asset/conversionAsset, I agree it is an improvement. I'll move forward with that for now.

Reply:
ah sorry, you have to set "primary" currency as "Fiat" in settings to see this. I did update the getConversionRate selector to return null instead of undefined and same result.